libc/calls/commandv.c

```diff
@@ -16,16 +16,20 @@
 │ TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR             │
 │ PERFORMANCE OF THIS SOFTWARE.                                                │
 ╚─────────────────────────────────────────────────────────────────────────────*/
-#include "libc/intrin/bits.h"
-#include "libc/intrin/safemacros.internal.h"
 #include "libc/calls/calls.h"
-#include "libc/intrin/strace.internal.h"
+#include "libc/calls/struct/stat.h"
+#include "libc/calls/syscall_support-nt.internal.h"
```

Comment:
I don't think this header is needed anymore: `#include "libc/calls/syscall_support-nt.internal.h"`